docs/library/ubluetooth.rst

```diff
@@ -181,6 +182,21 @@ Event Handling
             elif event == _IRQ_MTU_EXCHANGED:
                 # MTU exchange complete (either initiated by us or the remote device).
                 conn_handle, mtu = data
+            elif event == _IRQ_L2CAP_COC_ACCEPT:
+                # A new channel has been accepted.
+                chan_handle, = data
+            elif event == _IRQ_L2CAP_COC_CONNECT:
+                # A new channel is now connected (either as a result of connecting or accepting).
+                chan_handle, our_mtu, peer_mtu = data
+            elif event == _IRQ_L2CAP_COC_DISCONNECT:
+                # Existing channel has disconnected.
+                chan_handle, = data
+            elif event == _IRQ_L2CAP_COC_RECV:
```

Comment:
Do the names of events need to have "COC" in them, is it simpler to remove that part of the name, or is there scope to eventually add other L2CAP events which are not related to COC?

Also, could this be called a READ event to match other READ/WRITE named events?

Reply:
Summarising offline conversation here: L2CAP channels can either be fixed (reserved CIDs, connectionless) or dynamic (require a connection, i.e. CoC). Given that the listen/connect/disconnect methods imply connection-oriented, then they don't need to have CoC in their name. The send/recv methods could work on any type of channel.

I think if we write a "stream" abstraction around this, then read/write would make sense, but recv/send makes more sense for the low-level implementation.